SU-SWS/stanford_jumpstart_academic
# HSDO-829 added permissions for story_page

# READY FOR REVIEW

# Summary
- Added permissions for stanford_story_page

# Needed By (Date)
- 6/7

# Steps to Test

1. Checkout branch
2. enable [stanford_story_page](https://github.com/SU-SWS/stanford_story_page)
3. revert stanford_jsa_roles
4. check if permissions are applied for stanford_story_page node type.
